src/commands/templates/install.js

```diff
@@ -15,13 +15,38 @@ const { runScript } = require('../../lib/helper')
 const { writeObjectToPackageJson, readPackageJson, getNpmDependency, processNpmPackageSpec, TEMPLATE_PACKAGE_JSON_KEY } = require('../../lib/npm-helper')
 const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-app-templates:templates:install', { provider: 'debug' })
 
+// aio-lib-console-project-installation depenencies
+const path = require('path')
+const sdk = require('@adobe/aio-lib-console')
+const loadConfig = require('@adobe/aio-cli-lib-app-config')
+const templateHandler = require('@adobe/aio-lib-console-project-installation')
+
 class InstallCommand extends BaseCommand {
   async run () {
     const { args } = this.parse(InstallCommand)
     let templateName
 
     await runScript('npm', process.cwd(), ['install', args.path])
 
+    // Integrate with the template installation library
+    // Instantiate Adobe Developer Console SDK
+    await this.login()
+    const apiKey = 'aio-cli-console-auth'
+    const client = await sdk.init(this.accessToken, apiKey)
```

Comment:
I believe we agreed to move the @adobe/aio-lib-console client initialization into @adobe/aio-lib-console-project-installation

Reply:
Fixed ✅